virttest/qemu_devices/qcontainer.py

```diff
@@ -2674,7 +2674,9 @@ def chardev_define_by_params(self, chardev_id, params, file_name=None):
         if 'socket' in backend:  # tcp_socket & unix_socket
             chardev_param.update(
                 {'server': params.get('chardev_server', 'on'),
-                 'wait': params.get('chardev_wait', 'off')})
+                 'wait': params.get('chardev_wait', 'off'),
+                 'abstract': params.get('chardev_abstract', ''),
+                 'tight': params.get('chardev_tight', '')})
```

Comment:
tcp soscket does not support these two options:
```
-chardev socket,id=id[,host=host],port=port[,to=to][,ipv4=on|off][,ipv6=on|off][,nodelay=on|off][,reconnect=seconds]
         [,server=on|off][,wait=on|off][,telnet=on|off][,websocket=on|off][,reconnect=seconds][,mux=on|off]
         [,logfile=PATH][,logappend=on|off][,tls-creds=ID][,tls-authz=ID] (tcp)
```

Reply:
> tcp socket does not support these two options:
> 
> ```
> -chardev socket,id=id[,host=host],port=port[,to=to][,ipv4=on|off][,ipv6=on|off][,nodelay=on|off][,reconnect=seconds]
>          [,server=on|off][,wait=on|off][,telnet=on|off][,websocket=on|off][,reconnect=seconds][,mux=on|off]
>          [,logfile=PATH][,logappend=on|off][,tls-creds=ID][,tls-authz=ID] (tcp)
> ```

Thanks and update.